drafts/use-cases/0.intro.md

```diff
@@ -0,0 +1,93 @@
+# Uses cases for Hydra Core Vocabulary client
+
+Here you can find some of the use cases to consider while developing a reference client implementation and improve Hydra Core Vocabulary specification.
+
+To create a context of the whole task, these use-cases are related to the [EventAPI](https://github.com/lanthaler/EventApiDemo) and a calendar-like UI web application.
+
+Each scenario is build using several sections:
+
+- Story - a short _As a... I want... so..._ story introducing a general requirement
+- Usage - a pseudo-code (JavaScript/TypeScript like) of the imaginary client
+- Details - detailed description
+- Considerations - some additional considerations on several aspects
+
+## Assumptions
+
+For simplification, some assumptions are made so the main topic of each of the use-cases are not distracted with side details.
+
+These would be:
+
+- client-server communication uses JSON-LD RDF serialization using HTTP headers:
+  Content-Type: application/ld+json for server responses
+  Accept: application/ld+json client requests
+- JSON-LD context is described later on in this document and is referenced in HTTP header:
+  Link: </events/events.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"
+- main URL used is _/api_
+
+### JSON-LD context
+```javascript
+{
+    "hydra": "http://www.w3.org/ns/hydra/core#",
+    "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "owl": "http://www.w3.org/2002/07/owl#",
+    "schema": "http://schema.org/",
+    "title": {
+        "@id": "hydra:title",
+        "@type": "xsd:string"
+    },
+    "operations": {
+        "@id": "hydra:operation",
+        "@type": "hydra:Operation"
+    },
+    "httpMethod": {
```

Comment:
Do you want to propose to rename `method` to `httpMethod`? Otherwise I wouldn't remap it (things like plurals as in operations are fine though, even more so if you would also set `"@container": "@set"` as that would ensure values of such properties remain arrays when compacting the JSON-LD)